socorro/processor/mozilla_transform_rules.py

```diff
@@ -1092,7 +1092,8 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         '''addons is a list of tuples containing (extension, version)'''
         addons = processed_crash.get('addons', [])
 
-        for extension, version in addons:
+        for addon in addons:
+            extension, version = addon.split(':', 1)
```

Comment:
This makes me nervous. If there is ever an `addon` that doesn't contain a `:` you get a `ValueError`.
And, it USED to be that `addons` used to iterate over a list of tuples. Will it now ALWAYS be a string? ...for old and new crashes.